pwiz_tools/Skyline/Model/Lib/NistLibSpec.cs

```diff
@@ -829,7 +832,7 @@ private bool CreateCache(ILoadMonitor loader, IProgressStatus status, int percen
 
 
                     // Read until name line
-                    if (!line.StartsWith(NAME))
+                    if (!line.StartsWith(NAME, true, CultureInfo.InvariantCulture)) // Accept Name:, NAME:, nAmE: etc

```

Comment:
You could also do "line.StartsWith(NAME, StringComparison.InvariantCultureIgnoreCase)" which does exactly the same thing but might be more self-documenting

Reply:
Spiffy! Will do.